src/ProjectCanine/Views/SelectTestPage.xaml.cs

```diff
@@ -23,7 +23,7 @@ public SelectTestPage()
             if (item == null)
                 return;
 
-            await Navigation.PushAsync(new ItemDetailPage(new ItemDetailViewModel(item)));
+            await Navigation.PushAsync(new TestFrontPage(new ItemDetailViewModel(item)));
```

Comment:
Please also rename `ItemDetailViewModel` to `TestFrontPageViewModel`, to keep naming consistent.